pep-0554.rst

```diff
@@ -30,11 +30,11 @@ with new (basic) functionality for sharing data between interpreters.
 A Disclaimer about the GIL
```

Comment:
The build error is due to linking the now non-existent provisional section a few lines above this one; I imagine you'll want to elide that sentence entirely, since the PEP is no longer provisional.

Also, FYI, now that Sphinx is supported, you can use `:ref:` and ref target labels instead, which handle this better.

Reply:
Looks like you fixed it :+1: